facets_atlasmaker/montage.py

```diff
@@ -0,0 +1,191 @@
+"""Utilities to combine converted images into atlas and to create manifest."""
+import math
+import os
+from absl import logging
+from PIL import Image
+
+
+# Manifest Key names
+MANIFEST_IMAGE_NAME_KEY = 'image_name'
+MANIFEST_SOURCE_IMAGE_KEY = 'source image'
+MANIFEST_OFFSET_X_KEY = 'offset_x'
+MANIFEST_OFFSET_Y_KEY = 'offset_y'
+MANIFEST_IMAGE_SUCCESS_KEY = 'success'
```

Comment:
I like that there's a flag to indicate whether download was successful. This allows for easy querying with `jq` to find out what failed:

```
$ cat outputs/manifest.json | jq 'select(.success == false)'
```

Since success is by far the majority case, it would lead to a slightly smaller manifest to omit the `success` key and instead have an `error` key which only appears for the failures.

Besides switching from `success` to `error`, how hard would it be for the value of the error to be a string describing the kind of error? For example, the two kinds of errors I'm seeing are image is too big (`DecompressionBombWarning`) and 'cannot identify image file' (e.g. SVG and webm files).

Here's an example of the `DecompressionBombWarning`:

```
/Users/jimbo/facets-env/lib/python2.7/site-packages/PIL/Image.py:2514: DecompressionBombWarning: Image size (124090014 pixels) exceeds limit of 89478485 pixels, could be decompression bomb DOS attack.
  DecompressionBombWarning)
```

You don't have to implement this request, I'm just curious to get your assessment of how hard it might be to add a very brief reason for the `error`, e.g. `TOO_BIG`, `NON_IMAGE` or `OTHER`.